app/forms/steps/capital/property_form.rb

```diff
@@ -32,8 +32,6 @@ class PropertyForm < Steps::BaseFormObject
 
       validates_with CapitalAssessment::PropertyOwnershipValidator
 
-      validate :home_address, unless: :property_is_home_address?
-
       def persist!
         record.update(attributes)
       end
```

Comment:
I considered defaulting `is_home_address` to `no` if the question wasn't displayed but as it wasn't explicitly selected by the user i've left it to be `nil`